sdk/table/azure-table/azure/table/_table_client.py

```diff
@@ -0,0 +1,575 @@
+                """
+        timeout = kwargs.pop('timeout', None)
+        try:
+            _, identifiers = self._client.table.get_access_policy(
+                table=self.table_name,
+                timeout=timeout,
+                cls=return_headers_and_deserialized,
+                **kwargs)
+        except HttpResponseError as error:
+            process_storage_error(error)
+        return {s.id: s.access_policy or AccessPolicy() for s in identifiers}
+
+    @distributed_trace
+    def set_table_access_policy(self, signed_identifiers, **kwargs):
+        # type: (...) -> None
+        """Sets stored access policies for the table that may be used with Shared Access Signatures.
+
+                :param signed_identifiers:
+                :type signed_identifiers: {id,AccessPolicy}
+                :keyword callable cls: A custom type or function that will be passed the direct response
+                :return: None, or the result of cls(response)
+                :rtype: None
+                :raises: ~azure.core.exceptions.HttpResponseError
+                """
+        if len(signed_identifiers) > 5:
+            raise ValueError(
+                'Too many access policies provided. The server does not support setting '
+                'more than 5 access policies on a single resource.')
+        identifiers = []
+        for key, value in signed_identifiers.items():
+            if value:
+                value.start = serialize_iso(value.start)
+                value.expiry = serialize_iso(value.expiry)
+            identifiers.append(SignedIdentifier(id=key, access_policy=value))
+        signed_identifiers = identifiers  # type: ignore
+        try:
+            self._client.table.set_access_policy(
+                table=self.table_name,
+                table_acl=signed_identifiers or None,
+                **kwargs)
+        except HttpResponseError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def get_table_properties(
+            self,
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> "models.TableServiceProperties"
+        """Gets the properties of an account's Table service,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: TableServiceProperties, or the result of cls(response)
+        :rtype: ~azure.table.models.TableServiceProperties
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        timeout = kwargs.pop('timeout', None)
+        request_id_parameter = kwargs.pop('request_id_parameter', None)
+        try:
+            response = self._client.service.get_properties(
+                timeout=timeout,
+                request_id_parameter=request_id_parameter,
+                **kwargs)
+            return response
```

Comment:
Let's check what we're returning here, generally we would wrap this in an exposed model and not return the generated model.